internal/router/router.go

```diff
@@ -543,6 +544,8 @@ func (r *Router) serveBulkMetaGet(req *http.Request) (interface{}, error) {
 		return nil, errgo.WithCausef(nil, params.ErrBadRequest, "no ids specified in meta request")
 	}
 	delete(req.Form, "id")
+	ignoreAuth := req.Form.Get("ignore-auth") == "1"
```

Comment:
shouldn't we use parseBool here for consistency with the rest of the API (perhaps move it out of v4 into another package, or copy it)